pontoon/base/static/js/main.js

```diff
@@ -143,6 +143,38 @@ $(function () {
         ga('send', 'event', 'ajax', 'request', settings.url);
     });
 
+    /*
+     * Display Pontoon Add-On promotion, if:
+     *
+     * - Pontoon Add-On not installed
+     * - Page loaded on Firefox or Chrome (add-on not available for other browsers)
+     */
+    var isFirefox = navigator.userAgent.indexOf('Firefox') !== -1;
+    var isChrome = navigator.userAgent.indexOf('Chrome') !== -1;
+    var downloadLink = $('#addon-promotion').find('.get');
+    setTimeout(function () {
+        if (!window.PontoonAddon || !window.PontoonAddon.installed) {
+            if (!isFirefox && !isChrome) {
```

Comment:
Personally I would prefer the conditions like this
```js
let downloadLinkHref;
if (isFirefox) {
  downloadLinkHref = '...';
} else if (isChrome) {
  downloadLinkHref = '...';
}
if (downloadHref) {
  downloadLink.attr(...);
  $('body').addClass('addon-promotion-active');
}
```
Adding more browsers will be easier like this in the future.

Reply:
Happy to learn you're willing to add more browsers! 😆